services/movie_session.py

```diff
@@ -0,0 +1,80 @@
+from datetime import datetime, timedelta
+from django.core.exceptions import ObjectDoesNotExist
+from django.db.models import QuerySet
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    if isinstance(movie_show_time, str):
+        try:
+            show_time = datetime.strptime(movie_show_time, "%Y-%m-%d")
```

Comment:
The date parsing in `create_movie_session` assumes the date is in 'YYYY-MM-DD' format. If the time is also needed, consider using '%Y-%m-%d %H:%M:%S' for more precision.